### DISCUSSION TITLE
Error when trying to integrate web3Auth and Auth0

### DISCUSSION THREAD

**Discussion by RoniFinTech** [Web3Auth/Web3Auth#853] [Help]
Hi,

Following this guide:
https://web3auth.io/docs/guides/auth0

Getting this error:
```
File was processed with these loaders:
 * ./node_modules/react-scripts/node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
|   params.hardforks = Object.values(enums_1.Hardfork).map(name => ({
|     name,
>     block: name === enums_1.Hardfork.Chainstart ? 0 : config[forkMap[name]] ?? null
|   })).filter(fork => fork.block !== null);
|   if (config.terminalTotalDifficulty !== undefined) {
```

Any idea what could the issue be?

Thanks,
Roni

**Comment by rinzler99** [comment]
Hey, Thanks for reaching out to us. Can you please share the code snippet of the configuration? Also, try to delete `node_modules` and reinstall them.

**Reply by rinzler99** [reply]
And make sure you are using the latest `@web3auth/core` SDK.

**Reply by RoniFinTech** [reply]
Hi,

This is the snippet:
```
const web3authCore = new Web3AuthCore({
                    clientId,
                    chainConfig: { // this is ethereum chain config, change if other chain(Solana, Polygon)
                        chainNamespace: CHAIN_NAMESPACES.EIP155,
                        chainId: "0x1",
                        // rpcTarget: "https://mainnet.infura.io/v3/776218ac4734478c90191dde8cae483c",
                        blockExplorer: "https://etherscan.io/",
                        ticker: "ETH",
                        tickerName: "Ethereum"
                    },
                });
                const adapter = new OpenloginAdapter({
                    adapterSettings: {
                        network: "testnet",
                        // clientId,
                        uxMode: "redirect", // other option: popup
                        loginConfig: {
                            jwt: {
                                name: "dev-unik-web3auth",
                                verifier: "dev-unik-google-verifier",
                                typeOfLogin: "jwt",
                                clientId: "blahblah",
                            },
                        },
                    }
                });
                web3authCore.configureAdapter(adapter);
                await web3authCore.init();
                setWeb3auth(web3authCore)
                
                if (web3authCore.provider) {
                    setProvider(web3authCore.provider);
                }
```
                
I've tried removing node_moudles and reinstalling, but no help.

These are the versions I'm using:

```
"@web3auth/base": "^3.0.4",
"@web3auth/core": "^3.0.4",
"@web3auth/openlogin-adapter": "^3.0.4",
"web3": "^1.8.1"
```

**Comment by RoniFinTech** [comment]
any thoughts?

**Reply by rinzler99** [reply]
Hey Roni sorry for delay in reply. 
   Your configuration code for auth example with web3authCore constructor looks fine, I guess the problem is in react configuration. If you can share the repo of your project I can try running it in my local machine and try to resolve the issue.

**Reply by RoniFinTech** [reply]
> Hey Roni sorry for delay in reply. Your configuration code for auth example with web3authCore constructor looks fine, I guess the problem is in react configuration. If you can share the repo of your project I can try running it in my local machine and try to resolve the issue.

Thanks for the response.
This is a private repo... let me see how I can add you to the repo.

**Reply by RoniFinTech** [reply]
So I was able to overcome this issue and the problem was with the `config-override` file.
Looks like I got the file content from one of the guides and it was incomplete... the file here helped solved the issue:

https://web3auth.io/docs/troubleshooting/webpack-issues

I'm now having another issue unrelated to this and I will ask the question in a separate thread. 
Basically I initialize web3AuthCore in App.tsx and to share it among all files, I use a global store (useContext). Everything works fine until I want to store web3AuthCore in localstorage... need to figure out how to store custom classes with methods inside within localstorage to maintain state...
